sdk/react/upNext.js

```diff
@@ -123,19 +123,21 @@ class UpNext extends React.Component {
     if(this.isWithinShowUpNextBounds() && !this.props.upNextDismissed && upNextConfig.showUpNext === true && !this.props.ad && this.props.nextVideo != null) {
       var countdown = this.renderCountdownTimer();
       var upNextImage = (
+      <TouchableHighlight
+      style={styles.thumbnail}
+      accessible={true}
```

Comment:
Please add spaces here according to the code style guide.

Reply:
done